src/script/printer/tir/buffer.cc

```diff
@@ -126,6 +126,18 @@ ExprDoc BufferDecl(const tir::Buffer& buffer, const String& method, const Array<
                     /*args=*/args);
 }
 
+ExprDoc BufferAttn(const tir::Buffer& buffer, const ObjectPath& p, const Frame& frame,
+                   const IRDocsifier& d) {
+  Map<String, ExprDoc> attrs = BufferAttrs(buffer, p, frame, d);
+  Array<Doc> indices_doc;
+  for (String s : {"shape", "dtype"}) {
+    if (Optional<ExprDoc> doc = attrs.Get(s)) {
+      indices_doc.push_back(doc.value());
+    }
+  }
+  return TIR("Buffer")[indices_doc];
```

Comment:
Let's modernize it by using `->Call` instead!

Reply:
but `->Call` may generate `T.Buffer()` instead of `T.Buffer[]`?

Reply:
Yep, and it's more recommended that `T.Buffer[]`